applications/nrf_desktop/src/modules/ble_latency.c

```diff
@@ -45,7 +45,7 @@ static void security_timeout_fn(struct k_work *w)
 	 */
 	BUILD_ASSERT(CONFIG_BT_MAX_PAIRED == (CONFIG_BT_ID_MAX - 1));
 	BUILD_ASSERT(CONFIG_BT_MAX_CONN == 1);
-	BUILD_ASSERT(SECURITY_FAIL_TIMEOUT_MS > 0);
+	BUILD_ASSERT(CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S > 0);
```

Comment:
SECURITY_FAIL_TIMEOUT_MS  is used probably later on - why to use two things?

Reply:
SECURITY_FAIL_TIMEOUT_MS  is of type struct k_timeout_t, and cannot be used to compare.